blosc/schunk.c

```diff
@@ -686,6 +686,9 @@ int blosc2_schunk_update_chunk(blosc2_schunk *schunk, int nchunk, uint8_t *chunk
         if (frame->sframe) {
           schunk->cbytes -= cbytes_old;
         }
+        else if (cbytes_old >= cbytes) {
```

Comment:
Maybe it's better to replace this with:
```
else {  // Contiguous frame
  if (cbytes_old >= cbytes) {
    schunk->cbytes -= cbytes;
  }
}
```

In this way, we visually divide the `sframe` and the `cframe` sections. Then, within the `cframe` section, we will already evaluate the condition.

What do you think?

*The `else if` statement is recommended to use in situations like:*

```
if (year < 18) {
  // Do something
} else if (18 >= year && year < 25) {
  // Do something
} else {
  // Do something
}
```

Reply:
I see, changed in 9c3bf17a428af246f12b85fe2816bc0dd5177a98